### PR TITLE
semgrep 0.8.1

### PR DESCRIPTION
**Pull request by brendongo**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by brendongo** [comment]
Dependencies of semgrep changed so need to manually update PR.

File: Formula/semgrep.rb [hunk 1]
```diff
@@ -71,7 +75,7 @@ def install
         system "opam", "exec", "--", "make", "all"
         system "opam", "exec", "--", "make", "install"
         build_dir = "_build/default/bin/"
-        File.rename(build_dir + "main_semgrep_core.exe", build_dir + "semgrep-core")
+        File.rename(build_dir + "Main.exe", build_dir + "semgrep-core")
         bin.install build_dir + "semgrep-core"
```
Comment (bayandin) [inline Formula/semgrep.rb:79]: It looks like we could simplify it something like this:

```suggestion
        bin.install "_build/default/bin/Main.exe" => "semgrep-core"
```

Comment (bayandin) [reply Formula/semgrep.rb:79]: And could you please squash all the commits into one `semgrep 0.8.1`
Thanks!

**Review by bayandin — APPROVED** [review]
Thanks @brendongo for your contribution to Homebrew! 🎉 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**